### ISSUE TITLE
Cutorch illegal memory access was encountered

### ISSUE THREAD

**Issue by nomeyer** [jcjohnson/torch-rnn#32]
Hi,

First, thanks a lot for open sourcing this project :)

I don't know much at all about Torch, CUDA, etc. so I could be missing something. However some fruitless googling has led me here hoping you can help.

I'm training a model (3 layers, 512 rnn size, batch size 64, sequence length 128) on an AWS GPU instance using the AMI from https://github.com/brotchie/torch-ubuntu-gpu-ec2-install. After a little while, the training process crashes with the following error:

`Epoch 1.31 / 50, i = 12999 / 2086300, loss = 1.071470`
`Epoch 1.31 / 50, i = 13000 / 2086300, loss = 1.066751`
`val_loss =     2.1398722386657`
`THCudaCheck FAIL file=/tmp/luarocks_cutorch-scm-1-4337/cutorch/lib/THC/generic/THCStorage.cu line=48 error=77 : an illegal memory access was encountered
/usr/local/bin/luajit: cuda runtime error (77) : an illegal memory access was encountered at /tmp/luarocks_cutorch-scm-1-4337/cutorch/lib/THC/generic/THCStorage.c:147`

I'll try using the OpenCL backend but after reading this I don't look forward to it... https://github.com/jcjohnson/torch-rnn/issues/11

Thanks

**Comment by dgcrouse** [comment]
This appears to be related to another library and is more than a year old. Closing as unrelated and stale.
